packages/wct-mocha/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "wct-mocha",
+  "version": "1.0.0-pre.1",
+  "description": "Client-side library for testing web-components with Mocha.",
+  "main": "lib/index.js",
+  "files": [
+    "browser.js",
+    "lib/",
+    "LICENSE",
+    "README.md"
+  ],
+  "scripts": {
+    "build": "npm run build:compile && npm run build:browserjs",
+    "build:browserjs": "rollup -c",
+    "build:compile": "tsc",
+    "format": "find src -name \"*.ts\" | xargs clang-format --style=file -i",
+    "test": "npm run test:integration",
+    "test:integration": "cd test && npm run test:wct"
+  },
+  "keywords": [
+    "wct",
+    "mocha",
+    "test",
+    "web-components"
+  ],
+  "author": "The Polymer Project Authors",
+  "license": "BSD-3-Clause",
+  "bugs": {
+    "url": "https://github.com/Polymer/tools/issues"
+  },
+  "homepage": "https://github.com/Polymer/tools#readme",
```

Comment:
link to more specific url?

Reply:
Totes.

Updated to `"homepage": "https://github.com/Polymer/tools/tree/master/packages/web-component-tester",`